src/components/BlogPost/index.js

```diff
@@ -49,8 +49,27 @@ const BlogPost = ({ children, postData, ...props }) => {
   ]
   const formattedDate = monthArr[Month - 1] + " " + Day + ", " + Year
 
-  // html tag regex
-  const removeHtmlRegex = /(<([^>]+)>)/gi
+  function unescapeHTML(str) {
```

Comment:
Please define this method outside of component.

Reply:
Done.